### PR TITLE
Slightly improve script and add docs

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):
Documentation for `.tgz` build and ability to work from any directory.

### CODE REVIEW

**Comment by alesapin** [comment]
Need to test own instruction. I will wait for CI build.

File: docs/en/getting_started/install.md [hunk 1]
```diff
@@ -59,6 +59,36 @@ sudo yum install clickhouse-server clickhouse-client
 
 You can also download and install packages manually from here: <https://repo.yandex.ru/clickhouse/rpm/stable/x86_64>.
 
+### From tgz archives {#from-tgz-archives}
+
+It is recommended to use official pre-compiled `tgz` archives for all Linux distributions, where installation of `deb` or `rpm` packages is not possible.
+
+Required version can be dowloaded with `curl` or `wget` from repository <https://repo.yandex.ru/clickhouse/tgz/>.
```
Comment (alexey-milovidov) [inline docs/en/getting_started/install.md]: <https://repo.yandex.ru/clickhouse/tgz/>

Is it markdown?

File: docs/en/getting_started/install.md [hunk 2]
```diff
@@ -59,6 +59,36 @@ sudo yum install clickhouse-server clickhouse-client
...
+After that downloaded archives should be unpacked and installed with installation scripts. Example for the latest version:
+```bash
+export LATEST_VERSION=`curl https://api.github.com/repos/ClickHouse/ClickHouse/tags 2>/dev/null | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' | head -n 1`
+curl -O https://repo.yandex.ru/clickhouse/tgz/clickhouse-common-static-$LATEST_VERSION.tgz
```
Comment (alexey-milovidov) [inline docs/en/getting_started/install.md]: BTW `curl` is usually less available than `wget`.

File: docs/en/getting_started/install.md [hunk 3]
```diff
@@ -59,6 +59,36 @@ sudo yum install clickhouse-server clickhouse-client
...
+curl -O https://repo.yandex.ru/clickhouse/tgz/clickhouse-client-$LATEST_VERSION.tgz
+
+tar -xzvf clickhouse-common-static-$LATEST_VERSION.tgz
+sudo clickhouse-common-static-$LATEST_VERSION/install/doinst.sh
+
+tar -xzvf clickhouse-common-static-dbg-$LATEST_VERSION.tgz
+sudo clickhouse-common-static-dbg-$LATEST_VERSION/install/doinst.sh
+
+tar -xzvf clickhouse-server-$LATEST_VERSION.tgz
+sudo clickhouse-server-$LATEST_VERSION/install/predoinst.sh
```
Comment (alexey-milovidov) [inline docs/en/getting_started/install.md]: Just curious what is the purpose of separate `predoinst`?

**Review by alexey-milovidov — APPROVED** [review]

File: docs/en/getting_started/install.md (hunk 3 above)
Comment (alesapin) [reply docs/en/getting_started/install.md]: Don't know, `alien` creates this script.

File: docs/en/getting_started/install.md (hunk 2 above)
Comment (alesapin) [reply docs/en/getting_started/install.md]:
```
$ docker run --network=host -it archlinux bash
[root@click /]# wget
bash: wget: command not found
[root@click /]# curl
curl: try 'curl --help' for more information

$ docker run --network=host -it gentoo/stage3-amd64 bash
click / # wget
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
click / # curl
curl: try 'curl --help' or 'curl --manual' for more information

$ docker run --network=host -it vbatts/slackware bash
bash-4.3# wget
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
bash-4.3# curl
bash: curl: command not found
```
parity!

**Comment by alesapin** [comment]
Checked on gentoo under root user in docker. Works.

**Pull request merged**